oat-sa/extension-tao-testqti
# Feature/nex 183/e2e test for interactions

**Related to task:** 
https://oat-sa.atlassian.net/browse/NEX-183

**Basic tests created for:**
- Associate interaction 
- Choice interaction 
- Extended Text Interaction
- Hot Text Interaction
- Match Interaction
- Media Interaction 
- Order Interaction 
- Slider Interaction 
- Upload Interaction

**How to test**
- Go to tao/views/build/node_modules/@oat-sa/tao-e2e-runner
- Create cypress.env.json from cypress.env.sample.json and it should contain testFiles like this: "testFiles": "taoQtiTest/views/js/e2e/\*\*/interactions/**/*.spec.js",
- npm run test
- all tests should pass
- go for back office and check if items, tests and deliveries are good
